homeassistant/components/media_player/aquostv.py

```diff
@@ -40,7 +41,7 @@
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
     vol.Optional(CONF_USERNAME, default=DEFAULT_USERNAME): cv.string,
     vol.Optional(CONF_PASSWORD, default=DEFAULT_PASSWORD): cv.string,
-    vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.string,
+    vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
```

Comment:
We should remove the timeout option instead. Timeouts should not be configurable.

Reply:
Sounds good. Made a new commit. Also removed the retries option that was not being used in the code.